cdap-docs/build.sh

```diff
@@ -171,18 +174,23 @@ function build_docs_outer_level() {
 ################################################## current
 
 function build_all() {
+  echo "========================================================================="
   echo "Building GitHub Docs."
-  ./build.sh docs-github $ARG_2 $ARG_3
+  echo "========================================================================="
+  run_command docs-github-part $ARG_2 $ARG_3
```

Comment:
Not sure about environment carryover.
Google analytics: If we test for that, then are a ton of things we want to test and error check :) but I do not disagree with you.